addon/components/session-objective-list-item.hbs

```diff
@@ -42,7 +42,11 @@
       {{/if}}
     {{/each}}
   {{else if @editable}}
-    <button role="button" {{on "click" (fn @manageParents @objective)}}>
+    <button
+      role="button"
```

Comment:
Buttons don't need a button role, they are buttons, and therefor are buttons already. Also I can clearly see that you didn't write this originally and it was probably me, but please fix it because I can't bear for this button to be falsely accused of not being a button.

Reply:
removed.